packages/deployments/contracts/contracts/core/messaging/connectors/Connector.sol

```diff
@@ -36,7 +36,7 @@ abstract contract Connector is ProposedOwnable, MerkleTreeManager, ConnectorMana
 
   event SenderRemoved(address sender);
 
-  event Dispatch(bytes32 leaf, uint256 index, bytes32 root, bytes message);
+  event Dispatch(bytes32 leaf, uint256 index, bytes32 root, uint32 destinationDomain, bytes message);
```

Comment:
nit: we should make the domain indexed so we can find all messages to a specific domain